src/MfaVerifyController.js

```diff
@@ -10,7 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint complexity: [2, 21] max-statements: [2, 25] max-params: [2, 12]*/
+/* eslint complexity: [2, 21] max-statements: [2, 25] max-params: 0 */
 define([
```

Comment:
Guess if we move to imports, we can just get rid of this?

Reply:
correct.